mlflow/tracking/fluent.py

```diff
@@ -955,6 +940,7 @@ def search_runs(
     run_view_type=ViewType.ACTIVE_ONLY,
     max_results=SEARCH_MAX_RESULTS_PANDAS,
     order_by=None,
+    format="pandas",
```

Comment:
As lengthy as `output_format` is, I think that works well here (didn't realize that `format` was a builtin). Could we make that minor change?

Reply:
yup